entrypoint

```diff
@@ -14,8 +14,11 @@ EOF
 fi
 
 if [[ ! -z "$INPUT_TRUST" ]]; then
-  jbang trust add $INPUT_TRUST
```

Comment:
sorry that i missed this pr!

I can kind grok that using absolute path here can help but why not also set PATH?

Reply:
PATH was defined to find JBang.

As soon as you refer to JBang with an absolute path, you don't need the env var anymore.